BurnerB/Questioner-ADC-
# Bg hash passwords 163225601

#### What does this PR do?
Fixes hashing passwords and removing passwords from URL responses
#### Description of Task to be completed?
Add password hashing to user passwords
remove password and confirm password from responses


#### How should this be manually tested?
clone this repo
set up a virtual environment
pip install flask
run flask
PATCH THE URL /api/v1/user/auth/signin ON POSTMAN

Fll in the following fields in the body:
{
	 
	         "firstname":"",
	         "lastname":"",
	         "email":"",
	         "username":"",
	         "password":"",
	         "confirmpassword":""
}
Check out put
#### What are the relevant pivotal tracker stories?
#163225601

![remove password responses](https://user-images.githubusercontent.com/45785786/51190046-60ff3100-18f2-11e9-8a7f-4ead30d6b25b.png)
